schemas/com.endlessm.panel-extension.gschema.xml

```diff
@@ -1,5 +1,49 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <schemalist>
   <schema path="/com/endlessm/panel-extension/" id="com.endlessm.panel-extension" >
+    <key name="hot-corner-enabled" type="b">
```

Comment:
There's a similar key under `org.gnome.desktop.interface` named `enable-hot-corners`, it's probably better to either use it.

Reply:
Yes, thought of that but ended up going with the downstream key as this way we can still control whether hotcorner is enabled without affecting sections that don't enable the panel extension - but I can change if you prefer.

Reply:
Fwiw, on 3.8 we check both keys and only enable hotcorner if both are true but it has an issue where the hotcorner button menu to enable/disable hotcorners only changes the downstream key. On this PR I changed to only ever check the downstream key and ignore the upstream key.

Reply:
I think we discussed this before, and the outcome was: show the button if hot corner is on bottom-right, hide it otherwise.

Reply:
hmm, I think changing the behaviour here to hide the button if not on bottom-right would need some discussions involving the design team, as clicking the button works regardless of whether hotcorner targets are enabled or not (this on 3,8 and the PR here).

I would prefer not to change this behaviour here at all, if you think we should do it please open a separate task for that.

I thought of adding (not included in this PR) a new setting to allow advanced users to hide the button manually, but not doing it automatically...

Reply:
That said I think the best we can do here is to migrate hot-corner-enabled and set the value to the upstream key and use that to determine whether the hotcorner targets are enabled - but even that I am not so sure given the reasons I mentioned above regarding sessions not enabling the panel extension - but still willing to change this one.

Reply:
Updated.